src/cityreader/cityreader.py

```diff
@@ -60,12 +77,28 @@ def cityreader(cities=[]):
 
 # TODO Get latitude and longitude values from the user
 
+latitude_input1 = input("Enter a latitude coordinate. ")
+longitude_input1 = input("Enter a longitude coordinate. ")
+latitude_input2 = input("Enter a second latitude coordinate. ")
+longitude_input2 = input("Enter a second longitude coordinate. ")
+
 def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
   # within will hold the cities that fall within the specified region
-  within = []
-
+  lat1 = int(lat1)
```

Comment:
I wonder if setting these to be floats would be better or if it matters. If you needed to be very specific and enter in floats , you probably want to be able to account for that.